pycaret/internal/PycaretExperiment.py

```diff
@@ -5136,8 +5136,11 @@ def compare_models(
             display.display_monitor()
             display.display_master_display()
 
+        # pull() method retrieves metrics id to column the results
+        id_or_display_name = lambda x: x.id if self._ml_usecase == MLUsecase.TIME_SERIES else x.display_name
```

Comment:
Please turn this into a normal function (with `def`)

Reply:
refactor to `id_or_display_name` function